extensions/registry.md

```diff
@@ -0,0 +1,21 @@
+# Buildpack Registry API
+
+The following additional elements extend the [`buildpack.toml` data format](../buildpack.md#buildpacktoml-toml) section in the Buildpack Interface Specification.
+
+```toml
+[buildpack]
+
+  [publish.ignore]
+  files = ["<files to ignore>"]
+
+  [[publish.vendor]]
+  url = "<url to download>"
+  dir = "<directory to download>"
```

Comment:
Should this be `destination directory for download`?

Also, I don't understand what `files` means. Is the URL somehow unzipped/untarred?